relay-server/src/actors/global_config.rs

```diff
@@ -0,0 +1,105 @@
+use std::borrow::Cow;
+use std::sync::Arc;
+use std::time::Duration;
+
+use relay_dynamic_config::GlobalConfig;
+use relay_system::{Addr, Interface, Service};
+use reqwest::Method;
+use serde::{Deserialize, Serialize};
+
+use crate::actors::project_cache::ProjectCache;
+use crate::actors::upstream::{RequestPriority, SendQuery, UpstreamQuery, UpstreamRelay};
+
+// No messages are accepted for now.
+#[derive(Debug)]
+pub enum GlobalConfigMessage {}
```

Comment:
This is basically the name of the `Interface` - this is not a message per se. 
```suggestion
pub enum GlobalConfig {}
```